### ISSUE TITLE
Errors with Ruby 2.3

### ISSUE THREAD

**Issue by infojunkie** [meedan/lapis#14]
Getting this error:

```
        rake  db:migrate
rake aborted!
Bundler::GemRequireError: There was an error while trying to load the gem 'gem-licenses'.
Gem Load Error is: Licenses is not a module
Backtrace for gem load error is:
/home/kratib/.rvm/gems/ruby-2.3.0/gems/gem-licenses-0.2.0/lib/gem-licenses/version.rb:2:in `<module:Gem>'
/home/kratib/.rvm/gems/ruby-2.3.0/gems/gem-licenses-0.2.0/lib/gem-licenses/version.rb:1:in `<top (required)>'
/home/kratib/.rvm/gems/ruby-2.3.0/gems/gem-licenses-0.2.0/lib/gem_licenses.rb:2:in `require'
/home/kratib/.rvm/gems/ruby-2.3.0/gems/gem-licenses-0.2.0/lib/gem_licenses.rb:2:in `<top (required)>'
/home/kratib/.rvm/gems/ruby-2.3.0/gems/gem-licenses-0.2.0/lib/gem-licenses.rb:1:in `require'
/home/kratib/.rvm/gems/ruby-2.3.0/gems/gem-licenses-0.2.0/lib/gem-licenses.rb:1:in `<top (required)>'
/home/kratib/.rvm/gems/ruby-2.3.0/gems/bundler-1.12.5/lib/bundler/runtime.rb:86:in `require'
/home/kratib/.rvm/gems/ruby-2.3.0/gems/bundler-1.12.5/lib/bundler/runtime.rb:86:in `block (2 levels) in require'
/home/kratib/.rvm/gems/ruby-2.3.0/gems/bundler-1.12.5/lib/bundler/runtime.rb:81:in `each'
/home/kratib/.rvm/gems/ruby-2.3.0/gems/bundler-1.12.5/lib/bundler/runtime.rb:81:in `block in require'
/home/kratib/.rvm/gems/ruby-2.3.0/gems/bundler-1.12.5/lib/bundler/runtime.rb:70:in `each'
/home/kratib/.rvm/gems/ruby-2.3.0/gems/bundler-1.12.5/lib/bundler/runtime.rb:70:in `require'
/home/kratib/.rvm/gems/ruby-2.3.0/gems/bundler-1.12.5/lib/bundler.rb:102:in `require'
/media/hdd/src/meedan/karim/config/application.rb:7:in `<top (required)>'
/home/kratib/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
/home/kratib/.rvm/rubies/ruby-2.3.0/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
/media/hdd/src/meedan/karim/Rakefile:4:in `<top (required)>'
/home/kratib/.rvm/gems/ruby-2.3.0/gems/rake-11.1.2/lib/rake/rake_module.rb:28:in `load'
/home/kratib/.rvm/gems/ruby-2.3.0/gems/rake-11.1.2/lib/rake/rake_module.rb:28:in `load_rakefile'
/home/kratib/.rvm/gems/ruby-2.3.0/gems/rake-11.1.2/lib/rake/application.rb:689:in `raw_load_rakefile'
/home/kratib/.rvm/gems/ruby-2.3.0/gems/rake-11.1.2/lib/rake/application.rb:94:in `block in load_rakefile'
/home/kratib/.rvm/gems/ruby-2.3.0/gems/rake-11.1.2/lib/rake/application.rb:176:in `standard_exception_handling'
/home/kratib/.rvm/gems/ruby-2.3.0/gems/rake-11.1.2/lib/rake/application.rb:93:in `load_rakefile'
/home/kratib/.rvm/gems/ruby-2.3.0/gems/rake-11.1.2/lib/rake/application.rb:77:in `block in run'
/home/kratib/.rvm/gems/ruby-2.3.0/gems/rake-11.1.2/lib/rake/application.rb:176:in `standard_exception_handling'
/home/kratib/.rvm/gems/ruby-2.3.0/gems/rake-11.1.2/lib/rake/application.rb:75:in `run'
/home/kratib/.rvm/gems/ruby-2.3.0/gems/rake-11.1.2/bin/rake:33:in `<top (required)>'
/home/kratib/.rvm/gems/ruby-2.3.0/bin/rake:23:in `load'
/home/kratib/.rvm/gems/ruby-2.3.0/bin/rake:23:in `<main>'
```

**Comment by caiosba** [comment]
Fixed. The problem was with the `gem-licenses` gem. The problem was already filed in https://github.com/dblock/gem-licenses/issues/9. Basically, Rubygems 2.5 (the one for Ruby 2.3) defines a `Gem::Licenses` class, which conflicts with the one defined by `gem-licenses`. I forked the `gem-licenses` gem, renamed the class and set Lapis to use that fork instead of the official. I also submitted a PR to the official repo... if it gets merged, I'll make Lapis use that one again.
